kroma-validator/challenger.go

```diff
@@ -155,6 +163,14 @@ func (c *Challenger) InitConfig(ctx context.Context) error {
 		return fmt.Errorf("failed to initiate valpool config: %w", err)
 	}
 
+	cCtx, cCancel := context.WithTimeout(ctx, c.cfg.NetworkTimeout)
+	defer cCancel()
+	poolTerminationIndex, err := c.valpoolContract.TERMINATEOUTPUTINDEX(optsutils.NewSimpleCallOpts(cCtx))
+	if err != nil {
+		return fmt.Errorf("failed to get termination index: %w", err)
+	}
+	c.poolTerminationIndex = poolTerminationIndex
```

Comment:
I think we can move this up at the validator level and bring it down to the l2 output submitter and challenger.